.teamcity/settings.kts

```diff
@@ -438,9 +438,17 @@ object CheckCodeStyle : BuildType({
 				. "${'$'}NVM_DIR/nvm.sh" --no-use
 				nvm install
 
-				# Code style
-				FILES_TO_LINT=${'$'}(git diff --name-only --diff-filter=d refs/remotes/origin/master...HEAD | grep -E '^(client/|packages/|apps/)' | grep -E '\.[jt]sx?${'$'}' || exit 0)
+				# Find files to lint
+				if [[ "%teamcity.build.branch.is_default%" == "true" ]]; then
```

Comment:
Doc in https://www.jetbrains.com/help/teamcity/predefined-build-parameters.html#Branch-Related+Parameters